lib/logster/redis_store.rb

```diff
@@ -319,5 +409,15 @@ def protected_key
     def grouping_key
       @grouping_key ||= "__LOGSTER__GMAP"
     end
+
+    private
+
+    def register_rate_limit(severities, limit, duration, block)
```

Comment:
so the question is if we prefer the API 

ahh why not pass it in as a block to the private method?